src/component/common/Table/cells/ReviveArchivedFeatureCell/ReviveArchivedFeatureCell.tsx

```diff
@@ -0,0 +1,20 @@
+import { VFC } from 'react';
+import { ActionCell } from '../ActionCell/ActionCell';
+import { Undo } from '@mui/icons-material';
+import { IconButton } from '@mui/material';
+
+interface IReviveArchivedFeatureCell {
+    onRevive: any;
+}
+
+export const ReviveArchivedFeatureCell: VFC<IReviveArchivedFeatureCell> = ({
```

Comment:
This is so specific that it should be moved to `/component/archive/ArchiveTable/ReviveArchivedFeatureCell/ReviveArchivedFeatureCell.tsx`

We try to group elements that belong together so that we can cascade deletion of subcomponents/styles if we remove a component.